Testing/Data/UnitTest/ILL/PANTHER/001723.nxs.md5

```diff
@@ -0,0 +1 @@
+5640a0850f5afa4be7643c999f6756df
```

Comment:
Can you add a test in `LoadILLTOF` with this file, as it's the first one that is TOF?